labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -37,6 +42,15 @@ object OptionExercise02 {
    * to convert a possible numeric String (e.g. Some("12")) to an integer
    */
   def totalPeopleInRooms(rooms: Map[Int, Option[String]]): Int = {
-    error("Fix me")
+    def toOccupancy( o : Option[String] ):Int = o match {
+      case None => 0
+      case Some(s)=> 
+        try {
+          s.toInt
+        } catch {
+          case _:java.lang.NumberFormatException => 0
+        }
```

Comment:
That's tricky.  I see how it works, but it seems to me that it might be unnecessarily complex.